sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1728,6 +1741,147 @@ def plot_ild_reactions(self, subs=None):
 
         return PlotGrid(len(ildplots), 1, *ildplots)
 
+    def _solve_for_ild_shear(self):
+        """
+
+        Helper function for solve_for_ild_shear(). It takes the unsubstituted
+        copy of the load equation and uses it to calculate shear force equation.
+
+        """
+        x = self.variable
+        shear_force = -integrate(self._original_load, x)
+
+        return shear_force
+
+    def solve_for_ild_shear(self, dist, val, *reactions):
```

Comment:
It should rather be `distance` and `value`. This change should then also be done for `solve_for_ild_reactions()`